apps/condo/domains/common/utils/mutations.utils.js

```diff
@@ -43,7 +43,7 @@ function runMutation ({ action, mutation, variables, onCompleted, onError, onFin
                 let notificationContext = null
                 if (ErrorToFormFieldMsgMapping) {
                     const errors = []
-                    const errorString = `${e}`
+                    const errorString = `${JSON.stringify(e)}`
```

Comment:
![image](https://user-images.githubusercontent.com/1394025/129004692-07f4393d-5e7e-40a9-947a-2a768739b38c.png)

Reply:
made a check that the error has graphQLErrors in 65ef78a